src/mainwindow.cpp

```diff
@@ -6551,7 +6549,7 @@ void MainWindow::sendSelectedModsToSeparator_clicked()
 
       int newPriority = INT_MAX;
       bool foundSection = false;
-      for (auto mod : m_OrganizerCore.modsSortedByProfilePriority()) {
+      for (auto mod : m_OrganizerCore.modsSortedByProfilePriority(m_OrganizerCore.currentProfile())) {
```

Comment:
Doesn't calling without argument already use the currentProfile?

Reply:
Ah no you are doing that only for the modlist. Wouldn't it be easier to make it work like that for organizerCore as well?

Reply:
I thought about it but it was adding another default parameter and an extra check in `modsSortedByProfilePriority` which kind of feel redundant. Plus you always have access to `m_OrganizerCore.currentProfile()` if you are calling `m_OrganizerCore.modsSortedByProfilePriority()`.

Reply:
I guess so. Does it work properly for getting the order of other profiles? Like I'm not sure if the non active profile is loaded with all the needed stuff

Reply:
I did some quick testing with a Python module but in theory it should work properly:

- The priority stuff (`getPriorityMinimum`, `numRegularMods`, `modIndexByPriority`) depends on the `m_ModIndexByPriority` /  `m_NumRegularMods` which are set in `Profile::updateIndices()`.
- `Profile::updateIndices()` is called in `Profile::refreshModStatus()`.
- `Profile::refreshModStatus()` is called in all `Profile` constructors.

Reply:
That is nice hear, this could allow some cool profile sync stuff for plugins